rtf_vis_tool/src/Components/DivNode.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+//Basic Div Component repeatedly rendered in DivGraph.js
+const DivNode = (props) => {
+
+    // a standard styling for each node in the graph
+    const divNodeStyle = {
```

Comment:
can this go in a css?

Reply:
The only reason why this style cannot go in its own CSS is because of 3 styling properties on lines 9,10, and 11. These `top`, `left`, and `backgroundColor` stylings are unique in that they depend on `props`, which are passed into the component. These props can only be understood by JS code, not CSS code. Therefore, I need to keep this block of styling in JS.